pages/[word].tsx

```diff
@@ -22,61 +23,66 @@ export default function Word() {
 
   if (!wordName) return null
 
-  if (["XL", "LG"].includes(breakpoint.size)) {
-    return (
-      <DesktopLayout
-        theme={theme}
-        header={
-          <DesktopHeader
-            theme={theme}
-            searcher={<Searcher onSearch={(word) => router.push(`/${word}`)} />}
-            colorSwitcher={
-              <ColorSwitcher
-                themeName={themeName}
-                theme={theme}
-                setter={setTheme}
-              />
-            }
-            maxBodyWidth={800}
-          />
-        }
-        body={
-          <WordDefinition
-            word={wordName}
-            theme={theme}
-            onLoading={<Loading />}
-            onError={<div>{"Not found"}</div>}
-          />
-        }
-      />
-    )
-  }
-
   return (
-    <MobileLayout
-      header={(isHidden) => (
-        <MobileHeader
-          isShinkred={isHidden}
-          switcher={
-            <ColorSwitcher
+    <>
+      <Head>
+        <title>Dictionary - {wordName}</title>
```

Comment:
It would be better if you change the order of the word you are looking for by the page title, since the most important thing is not where you are looking, but what you are looking for.